fmatch/matcher.py

```diff
@@ -72,6 +72,7 @@ def get_uuid_by_metadata(
         index: str = None,
         lookback_date: datetime = None,
         lookback_size: int = 10000,
+        timestamp_field: str = "timestamp",
```

Comment:
I know this string literal was already in place somewhere else, but is it worth it to extract a constant?